protocol/bc/transaction.go

```diff
@@ -232,6 +239,23 @@ func (tx *Tx) WitnessHash() (hash Hash, err error) {
 	return hash, nil
 }
 
+func (tx *TxData) commonWitnessHash() (h Hash, err error) {
```

Comment:
It is today, but it may not be in the future. Added the suggested comment to `writeCommonWitness` instead.